src/ensembl/robots-txt.config.js

```diff
@@ -1,4 +1,4 @@
 module.exports = {
   policy: [{ userAgent: '*', disallow: '/' }],
-  host: 'http://2020.ensembl.org'
+  host: process.env.DEPLOYENV === 'prod' ? '2020.ensembl.org' : process.env.DEPLOYENV + '-2020.ensembl.org'
```

Comment:
probably should account for undefined as well? ;-)

Reply:
According to this line
  - case "${CI_COMMIT_REF_NAME}" in dev) DEPLOYENV="staging" ;; master) DEPLOYENV="prod" ;; cicd-test) DEPLOYENV="staging" ;; *) DEPLOYENV="dev" ;; esac
it is

Reply:
`probably should account for undefined as well?`

I've done that now so that the default is 2020.ensembl.org